README.md

```diff
@@ -322,6 +325,114 @@ See above for creating the table
 }
 ```
 
+## Dead Letter Table 
+
```

Comment:
Example config with Dead Letter will be very useful

Reply:
Docs can always be improved.  I'll try to take another stab at this. 

This is a big feature with a somewhat clunky coinfig API due to config visibility rules in Kafka Connect, so more docs/examples certainly help.